config/locales/client.en.yml

```diff
@@ -2105,6 +2105,9 @@ en:
       cannot_see_mention:
         category: "You mentioned %{username} but they won't be notified because they do not have access to this category. You will need to add them to a group that has access to this category."
         private: "You mentioned %{username} but they won't be notified because they are unable to see this personal message. You will need to invite them to this PM."
+      here_mention:
+        one: "By mentioning <b>@here</b>, you are about to notify %{count} user – are you sure?"
```

Comment:
We have to pass `@here` from `SiteSetting,mention_name`. Otherwise, the translation becomes inaccurate if the site setting has been changed from the default.